cmd/crane/main.go

```diff
@@ -22,8 +22,8 @@ import (
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "d8s",
-	Short: `D8s (Short for "dockerless") is a tool for managing container images`,
+	Use:   "crane",
+	Short: "Crane is a tool for managing container images, locally or in a registry",
```

Comment:
"locally or in a registry" seems kind of redundant? Could add a feature list under the `Long` description

Reply:
Removed.